application/src/main/resources/ecc.yml

```diff
@@ -84,6 +84,13 @@ repair:
   interval:
     time: 7
     unit: days
+
+  ##
+  ## How long from current time should the last repaired time be simulated/assumed for new tables.
+  ##
+  initial_delay:
+    time: 1
+    unit: hours
```

Comment:
This is a bad default, since the legacy value is one day, the default for this new property should of course reflect that; 1 day.